exporter.go

```diff
@@ -333,6 +333,10 @@ func (b *Exporter) handleEvent(event Event) {
 	metricName := ""
 	prometheusLabels := event.Labels()
 	if present {
+		if mapping.Name == "" {
+			log.Debugf("The mapping for match \"%v\" generates an empty metric name.", mapping.Match)
```

Comment:
Please also add a metric to track this. On line 255 we are already (mis)using the `eventStats` counter to track negative increments, but I think it would be better if you set up a new metric vector for event errors, and also changed the negative-counter to use it.

Reply:
Also, please also add `event.MetricName()` to the log line, this will make it much easier to track down what input causes this.